### PR TITLE
"Apps" uncovered in details, as a distributed services

### PR DESCRIPTION
**Pull request by dr-orlovsky**

### CODE REVIEW

**Comment by akhavr** [comment]
"We develop open source platform for uncensorable uncensorable " ? :)

File: gtecblockchaincontest.md [hunk 1]
```diff
@@ -4,9 +4,10 @@
 ## Describe your project and the problem it solves
 
 Internet services suffer from centralization which hinders innovation.
-We develop open source platform for uncensorable communication and
-apps.  Our first apps are Social Network, Group Decision (on
-blockchain) and Finances.
+We develop open source platform for uncensorable uncensorable 
+and decentralised services, such as social networks or messenging, which
+would be available in some sort of decentralized marketplace ("app store").
+For the beginning we'll start with Social Network, Group Decision and Finances.
```
Comment (UkolovaOlga) [inline gtecblockchaincontest.md:10]: We develop an open source platform for uncensorable and decentralised
services, such as social networks or messaging, which would be available
on some sort of decentralized  marketplace ("app store"). 
As a first step, we'll start with Social Network, Group Decision and Fincances.

**Review by UkolovaOlga — COMMENTED** [review]
Typos have been fixed, PR is ready for merging @akhavr

File: gtecblockchaincontest.md (hunk 1 above)
Comment (akhavr) [reply gtecblockchaincontest.md:10]: @UkolovaOlga necromerging? ;)

**Pull request merged**